tests/Domain/DomainEventTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Domain;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+abstract class DomainEventTestCase extends KernelTestCase
```

Comment:
Require the framework for domain tests is a sign that something isn't totally decoupled

Reply:
Should I mock (de)normalizer ? In this case, tests aren't accurate anymore because of mocked returns. Or maybe this tests are useless now because we moved serialization to external class

Reply:
I'd didn't explain myself well enough sorry.
In the prev version events were having serializer as dependency. This isn't the case anymore. The serializer tests are infrastructure and needs to be moved to infrastructure. In the domain layer we shouldn't require the symfony kernel for test at all as it must be framework agnostic

Reply:
BTW most of this serializer tests are probably covered by the e2e tests or the application ones

Reply:
So removing these tests is fine?

Reply:
Yes, no longer apply to domain should be ported to infrastructure